username_api.py

```diff
@@ -19,6 +25,42 @@
 # Discussion: github.com/manu-chroma/username-availability-checker/issues/79
 session = r.Session()
 
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter('[%(levelname)s|%(filename)s:%(lineno)s] '
+                              '%(asctime)s %(message)s')
+handler.setLevel(logging.WARNING)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+DEBUG = False
+
+
+def log_response(logger, msg, response, *args, **kwargs):
```

Comment:
``logger`` is a global variable, no need to pass a func param

Reply:
Just in case, it's for several loggers.
Additionally, it can be reused by others.